integration/feature/scoverage/repo/build.sc

```diff
@@ -0,0 +1,174 @@
+// Reproduction of issue https://github.com/com-lihaoyi/mill/issues/2579
```

Comment:
Can we minimize this? The external plugins will cause headaches in future if we let them get scattered throughout the codebase, but even ignoring those this test suite  is way larger than what I would expect is needed to reproduce the two issues.

Reply:
Sure, this is WIP, to find the issue soon.

Reply:
I wonder whether we should add another section "repro" to explicitly keep reproducers for known issues.